tests/concepts/test_View.cpp

```diff
@@ -27,4 +32,56 @@ TEST(concepts, View)
     static_assert(Kokkos::utils::concepts::View<view_2d_t>);
 }
 
+//! @test Check @ref Kokkos::utils::concepts::ViewOfInstanceOf.
+TEST(KokkosTraits, is_view_of_instance_of)
+{
+    using Kokkos::utils::concepts::ViewOfInstanceOf;
+
+    static_assert(   ViewOfInstanceOf<Kokkos::View<std::vector<int> , execution_space>, std::vector>);
+    static_assert(   ViewOfInstanceOf<Kokkos::View<std::vector<int>*, execution_space>, std::vector>);
+    static_assert( ! ViewOfInstanceOf<Kokkos::View<std::vector<int>*, execution_space>, std::tuple>);
+}
+
+//! @test Check @ref Kokkos::utils::concepts::ViewOf.
+TEST(KokkosTraits, is_view_of)
+{
+    using Kokkos::utils::concepts::ViewOf;
+
+    static_assert(   ViewOf<Kokkos::View<double       , execution_space>, double>);
+    static_assert(   ViewOf<Kokkos::View<double[5]    , execution_space>, double>);
+    static_assert(   ViewOf<Kokkos::View<double*      , execution_space>, double>);
+    static_assert(   ViewOf<Kokkos::View<double**     , execution_space>, double>);
+    static_assert(   ViewOf<Kokkos::View<const double*, execution_space>, const double>);
+    static_assert( ! ViewOf<Kokkos::View<const double*, execution_space>, double>);
+    static_assert( ! ViewOf<Kokkos::View<int          , execution_space>, double>);
+    static_assert( ! ViewOf<             double                         , double>);
+}
+
+//! @test Check the behavior of @ref Kokkos::utils::concepts::ViewOfRank.
+TEST(KokkosTraits,is_view_of_rank)
+{
+    using Kokkos::utils::concepts::ViewOfRank;
+
+    using view_rank_one_t = Kokkos::View<int* , execution_space>;
+    using view_rank_two_t = Kokkos::View<int**, execution_space>;
```

Comment:
```suggestion
    using int_view_t = Kokkos::View<int* , execution_space>;
    using int_2d_view_t = Kokkos::View<int**, execution_space>;
```

Reply:
Also, let's check the case of the zero-rank view. To be sure :)

Reply:
Done :cactus: